CMakeLists.txt

```diff
@@ -114,6 +114,9 @@ ENDIF()
 
 SET(CMAKE_C_STANDARD 11)
 ADD_DEFINITIONS(-DREDISMODULE_EXPERIMENTAL_API)
+if(REDISAI_LITE)
```

Comment:
`LITE`

Reply:
@rafie here https://github.com/RedisAI/RedisAI/pull/715#discussion_r624452581 you just wrote the other way around